src/status_im/contexts/preview/quo/banners/alert_banner.cljs

```diff
@@ -0,0 +1,38 @@
+(ns status-im.contexts.preview.quo.banners.alert-banner
```

Comment:
can you also move this to
`status-im.contexts.preview.status_im.banners.alert-banner` ?